Jenkinsfile

```diff
@@ -5,7 +5,8 @@ properties([
     pipelineTriggers([cron('H 5 * * 3')]),
 ])
 
-node('linux') {
+// 'linux-arm64' is now common between ci.jenkins.io and infra.ci.jenkins.io for VM "Docker Linux ARM64"
```

Comment:
```suggestion
```

(nit) I would remove the comment and update https://github.com/jenkins-infra/documentation/blob/main/ci.adoc instead

Reply:
what about both ?
leaving the comment AND editing the doc ?

Reply:
I don't see what is the point of the comment: the usual pipeline reader/writer is the developer of the project. THe comment here only drown them under information with no value.


Eventually we could link to the doc, but the current syntax does not provide any value